CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 - Many missing keys
 - Check if ScrollLock is toggled
 - `bind_all`
+- MousewheelUp and MousewheelDown
```

Comment:
0.6.0 was already released, so these should go to a new 0.7.0 section.

Reply:
Done